app/services/hyrax/analytics/ga4.rb

```diff
@@ -0,0 +1,204 @@
+# frozen_string_literal: true
```

Comment:
is there expectation of devergence or should we be creating a concern to house a bunch of this functionality that GA and GA4 can share?

Reply:
I was leaving the old GA code untouched in case this is in use while both services are active.

Reply:
that makes sense to me since its likely GA3 will be dead code before too long